### DISCUSSION TITLE
Welcome to chrome Discussions!

### DISCUSSION THREAD

**Discussion by joelgriffith** [browserless/chrome#2235] [Announcements]
## Welcome 👋 !
We value your feedback about browserless, and want to know what your thoughts and ideas are! Discussions seem like a great fit for this, so we'd love it if you'd:
  * Ask questions about what you're working on.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kerematam** [comment]
Hey! Thank you for this great tool.

I was trying older versions of chrome (by just changing deb files in Dockerfile.)
List here: https://www.slimjet.com/chrome/google-chrome-old-version.php

The oldest version could run is [86.0.4240.75](https://www.slimjet.com/chrome/download-chrome.php?file=files%2F86.0.4240.75%2Fgoogle-chrome-stable_current_amd64.deb). The beyond that (i mean older ones), i could not make it work. I only tested on debugger UI and it did not respond to my start (button click).

Any idea? Also, i am open to other approaches for testing older versions of chrome.

Thanks in advance!

**Comment by joelgriffith** [comment]
Are you intentionally trying to use legacy versions of Chrome? Sorry, just confused on what your end goal is :)
